cob_camera_sensors/package.xml

```diff
@@ -8,28 +8,22 @@
   <url type="website">http://ros.org/wiki/cob_camera_sensors</url>
   <!-- <url type="bugtracker"></url> -->
 
-  <maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer> 
+  <maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
   <author email="jsf@ipa.fhg.de">Jan Fischer</author>
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>cob_vision_utils</build_depend>
-  <build_depend>cv_bridge</build_depend>
-  <build_depend>image_transport</build_depend>
-  <build_depend>message_filters</build_depend>
   <build_depend>message_generation</build_depend>
-  <build_depend>polled_camera</build_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>sensor_msgs</build_depend>
-  
-  <run_depend>cob_vision_utils</run_depend>
-  <run_depend>cv_bridge</run_depend>
-  <run_depend>image_transport</run_depend>
-  <run_depend>message_filters</run_depend>
-  <run_depend>message_runtime</run_depend>
-  <run_depend>polled_camera</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>rospy</run_depend>
-  <run_depend>sensor_msgs</run_depend>
+  <exec_depend>message_runtime</exec_depend>
+
+  <depend>cob_vision_utils</depend>
+  <depend>cv_bridge</depend>
```

Comment:
Pulls this in `opencv`? If yes, this is fine, otherwise we might need to add opencv due to [this line](https://github.com/ipa320/cob_driver/blob/indigo_dev/cob_camera_sensors/ros/src/all_camera_viewer.cpp#L81)